package.json

```diff
@@ -77,6 +78,11 @@
     "oracledb": "5.3.0",
     "papaparse": "5.3.1",
     "pg": "8.7.1",
+    "pino": "git://github.com/rokku-x/pino.git#95b1392dcb1feefec71469434da594a3f557ff11",
```

Comment:
Obviously, we will have to use the normal version of pino when the compatibility with pkg is fixed.

Reply:
Yes, I'm watching these repos